iOS/Flows/Conversation/Cells/ConversationMessageCellLayoutAttributes.swift

```diff
@@ -13,7 +13,7 @@ class ConversationMessageCellLayoutAttributes: UICollectionViewLayoutAttributes
     /// If true, then text should be shown in the speech bubble.
     var shouldShowText: Bool = true
     /// The color of the background of the cell.
-    var backgroundColor: Color = .lightGray
+    var backgroundColor: Color = .white
```

Comment:
White is the default background color for all cells now. We get shades of gray by decreasing brightness.